metadata.display.yaml

```diff
@@ -84,6 +86,16 @@ spec:
         primary_instance:
           name: primary_instance
           title: Primary Instance
+          properties:
+            database_flags:
+              name: database_flags
+              title: database_flags
+              subtext: Customizable configuration settings that fine-tune database behavior, performance, and features to suit application needs.
```

Comment:
Do we have some documentation for configuring database_flags? We can include a link here.